tests/testthat/test-epi_shift_internal.R

```diff
@@ -0,0 +1,65 @@
+library(dplyr)
+library(epiprocess)
+library(parsnip)
+library(workflows)
+
+# Random generated dataset
+set.seed(100)
+x <- tibble(geo_value = rep("nowhere",200),
+              time_value = as.Date("2021-01-01") + 0:199,
+              case_rate = rpois(100,20) + 1:200,
+              death_rate = rpois(100,10) + 1:200) %>%
+  as_epi_df()
+
+slm_fit <- function(recipe, data = x) {
+  workflow() %>%
+    add_recipe(recipe) %>%
+    add_model(linear_reg()) %>%
+    fit(data = data)
+}
```

Comment:
I added tests to check for how the function works (ex: right number of predictors when doing stepping). I still don't quite get how I would check a function for fit aside from making sure it does the right number of steps and the linear function itself is right (rather than the outputs).

I also took out the randomness.